src/main/java/BasicJavaHomeWork/Exercise_2.java

```diff
@@ -0,0 +1,25 @@
+package BasicJavaHomeWork;
+
+import java.util.Scanner;
+
+public class Exercise_2 {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        int num = in.nextInt();
+        int i = num % 3;
```

Comment:
if decided to use variables, give them accordingly names

Reply:
done